int16 move_num
RobotCoord tom_coord
RobotCoord jerry_coord